app/src/main/java/com/ncnf/utilities/Helpers.java

```diff
@@ -0,0 +1,14 @@
+package com.ncnf.utilities;
+
+import java.util.concurrent.CompletableFuture;
+
+public class Helpers {
+
+    /**
+     * Method to combine the result of multiple CompletableFuture
+     */
+    public static CompletableFuture<Boolean> combine(CompletableFuture<Boolean> u1, CompletableFuture<Boolean> u2) {
```

Comment:
That might need to be in a specific db class, either rename the class it to make its purpose clearer, or move it to the db related package

Reply:
I moved it here because this is just a generic operation performed on Future so it is not related to the DB.